log_functions.py

```diff
@@ -47,3 +54,28 @@ def log_verbosity_to_logging(verbosity: str):
 def handle_error_and_exit(error: typing.Union[str, Exception]):
     logging.error(str(error))
     sys.exit(1)
+
+
+def send_email(error: str, date_time: str = None) -> None:
+    smtp_conn = smtplib.SMTP(config.notification.smtp_server)
+    smtp_conn.login(config.notification.smtp_username, config.notification.smtp_password)
+
+    msg = MIMEMultipart()
+    msg["Subject"] = config.notification.email_subject
+    msg["From"] = config.notification.email_sender
+    msg["To"] = ", ".join(config.notification.email_recipients)
+    msg.preamble = error
+
+    if date_time:
+        error = f"{date_time}: {error}"
+
+    msg.attach(MIMEText(error, "plain"))
```

Comment:
Why use MIMEMultipart + MIMEText instead of `email.message.EmailMessage` ? That one seems to have more convenient api...?
https://docs.python.org/3/library/email.examples.html